docs/modules/operation/pages/quick-start/users.adoc

```diff
@@ -28,20 +28,34 @@ NOTE: Admin users can xref:deep-dive/admin/housekeeping/introduction.adoc#disabl
 
 After logging in for the first time, make sure to change the default `admin` account password to a secure one:
 
-. Click the gear icon at the top-right of the screen.
-. Choose menu:OpenNMS System[Configure Users, Groups and On-Call Roles > Configure Users].
-. Click *Modify* beside the admin user.
-. In the *User Password* area, click *Reset Password*.
+. Click the *Gear* icon at the top-right of the screen.
+. Choose menu:OpenNMS System[Configure Users, Groups and On-Call Roles, Configure Users].
+. Click *Modify* beside the `admin` user.
+. In the User Password area, click *Reset Password*.
 . Update the password and click *OK*.
 . Click *Finish* at the bottom of the *Modify User* screen to save changes.
 
-WARNING: You cannot use angle brackets (<>), single (') and double quotation marks ("), or the ampersand symbol (&) in the user ID.
-
-We recommend that you do not use the default `admin` user account, but instead xref:deep-dive/user-management/user-config.adoc#ga-user-config [create specific users] with the admin role and/or other permissions.
+We recommend that you do not use the default `admin` user account, but instead xref:deep-dive/user-management/user-config.adoc#ga-user-config[create specific users] with the Admin role and other permissions.
 This helps to keep track of who has performed tasks such as clearing alarms or creating notifications.
-You should still change the password of the admin user to be more secure.
+
+For the purposes of this Quick Start guide, create one new user account for yourself and assign it the Admin role.
+Include your email address and any other relevant contact information for notification routing:
+
```

Comment:
Since we're including the configuration, let's just have them create the user here.

Reply:
This may be a more significant change, so I'll do it locally and push it. :)